samples/outbox/store-and-forward/Version_5/Sender/app.config

```diff
@@ -1,17 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@SzymonPobiega @weralabaj and sorry it is not immediately obvious. it is the same for me. i usually need to think about it.  i am working to have some extra metadata in the yaml header so we can capture "thing this page is targeting" and generate some common header html that explains it